content/en/serverless/distributed_tracing/serverless_trace_propagation.md

```diff
@@ -1,180 +0,0 @@
----
-title: Serverless Trace Propagation
-kind: documentation
-further_reading:
-- link: "/tracing/"
-  tag: "Documentation"
-  text: "Explore Datadog APM"
-- link: "/tracing/trace_search_and_analytics/#live-search-for-15-minutes"
-  tag: "Documentation"
-  text: "Live Search"
-- link: "https://www.datadoghq.com/blog/aws-serverless-tracing-datadog-apm/"
```

Comment:
I might have missed this, but I couldn't find this transferred to the new page further reading.